frontend/src/metabase-types/api/mocks/parameters.ts

```diff
@@ -9,7 +9,7 @@ export const createMockParameter = (opts?: Partial<Parameter>): Parameter => ({
 });
 
 export const createMockParameterSourceOptions = (
```

Comment:
There are still lots of instances where things haven't been renamed.

Reply:
that FE and I really don't want to mess it up because @ranquild  might be working on another branch based on this.

It's better for him to change it.

Reply:
Shouldn't it be `ParameterValuesSourceConfig`?

Reply:
tbh `ParameterValuesSourceConfig` seems like a complete nonsense to me, but I'm happy with that if BE fols are ok with that

Reply:
@ranquild I know you deleted your comment, but on the backend the schema for this value is called `ValuesSourceConfig`, not `ParameterValuesSourceConfig` Perhaps the names should be mirrored FE on BE.